### ISSUE TITLE
new computer no sounds play/load - net::ERR_CONTENT_DECODING_FAILED

### ISSUE THREAD

**Issue by rz-1** [Hanabi-Live/hanabi-live#2929]
**Describe the bug**
I logged into the site from a different computer and none of the sounds are playing/loading. Tried first in chrome, then brave browser.

errors from the console in chrome:
```
GET https://hanab.live/public/sounds/tone.mp3 net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-blind1.mp3:1 
        
        
       GET https://hanab.live/public/sounds/turn-blind1.mp3 net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-fail1.mp3:1 
        
        
       GET https://hanab.live/public/sounds/turn-fail1.mp3 net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-other.mp3:1 
        
        
       GET https://hanab.live/public/sounds/turn-other.mp3 net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-us.mp3:1 
        
        
       GET https://hanab.live/public/sounds/turn-us.mp3 net::ERR_CONTENT_DECODING_FAILED
```

when going to settings and pressing the test sound button
```
GET https://hanab.live/public/sounds/turn_us.mp3 net::ERR_CONTENT_DECODING_FAILED
playerSettings.ts:55 Failed to play the test sound: DOMException: Failed to load because no supported source was found.
```

errors from the console in brave:
```

public/sounds/tone.mp3:1 
        
        
       Failed to load resource: net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-blind1.mp3:1 
        
        
       Failed to load resource: net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-other.mp3:1 
        
        
       Failed to load resource: net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-fail1.mp3:1 
        
        
       Failed to load resource: net::ERR_CONTENT_DECODING_FAILED
public/sounds/turn-us.mp3:1 
        
        
       Failed to load resource: net::ERR_CONTENT_DECODING_FAILED
```

**Screenshots**
chrome console errors:
![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/9b0d952e-201f-46c2-8279-fe40d7871be6)

brave console errors:
![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/0dab8be5-e71a-4b55-a1aa-10866e6f91e4)

**Comment by Zamiell** [comment]
ERR_CONTENT_DECODING_FAILED is a new error for us i think.
it looks like it might not be our fault:
https://kinsta.com/knowledgebase/err_content_decoding_failed/
does the errors only happen on the new computer? does that computer have a firewall that could be breaking the traffic? or maybe it is passing through a work firewall/proxy server, etc.

**Comment by rz-1** [comment]
I tried everything suggested by that site (and others) yesterday and nothing has helped. Both computers are on the same network, plugged into ethernet ports on the same router. So I don't suspect anything at the router level, but I'll check. 

I'm able to hear sounds on other websites from this browser (youtube, etc), I'm guessing it has something to do with mp3 files, or these specific mp3 files. As those are the only files I cannot access. I've even tried to open the files directly (https://hanab.live/public/sounds/tone.mp3) and get the same error. 

![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/12c65689-9e62-4227-b8b9-eb4a2f948897)

**Comment by Zamiell** [comment]
its a local firewall on your new computer then, something like ZoneAlarm or Norton Protection.
reformat it with a fresh vanilla windows 10 and absolutely nothing else, and then see if you get the error.

**Comment by rz-1** [comment]
Way ahead of you. I just finished a format and reinstall of Windows 11, and most sounds are working again as far as I can tell. The sound that plays in Settings when clicking the Test button is not working. Looking at the network tab, it says it downloaded, but it tried to download it again and failed. I'm not sure why that one doesn't work.

![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/6bf0f26f-5829-4ef6-9593-a0efc1dfa4e4)

![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/eb25f4f3-004b-4004-8426-80c4f6a77bbb)

**Comment by rz-1** [comment]
I was just spectating a game, and looks like one other sound is not playing.

![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/d510f32d-46ad-418a-856f-c52d2e1ae67e)

**Comment by Zamiell** [comment]
> I think this error is because the file name pointing to the old version

should be fixed now, thanks

**Comment by Zamiell** [comment]
as for the ERR_CONTENT_DECODING_FAILED, given that you are the only one reporting the error, the problem is likely on your side.

did you reformat with a vanilla windows, or some modified version from your laptop manufacturer?
if you used a vanilla windows, then i should in theory be able to replicate the error simply by creating a new VM with vanilla windows 11 and then surfing to hanabi live, should i do that and see if i can replicate the error?

**Comment by rz-1** [comment]
> as for the ERR_CONTENT_DECODING_FAILED, given that you are the only one reporting the error, the problem is likely on your side.

That's probably the case, I can go through these steps again to see if it resolves https://kinsta.com/knowledgebase/err_content_decoding_failed/ . Very odd that it is only happening for a subset of mp3 files now.


> did you reformat with a vanilla windows, or some modified version from your laptop manufacturer? if you used a vanilla windows, then i should in theory be able to replicate the error simply by creating a new VM with vanilla windows 11 and then surfing to hanabi live. should i do that and see if i can replicate the error?

yes, I created a flash drive of Windows 11 from the Microsoft website. I'll have some time later this evening to try the above steps to fix, you can wait til then or test now. Up to you.

**Comment by rz-1** [comment]
I figured it out. It was the NordVPN Web Protection feature. As soon as I disabled that the sounds load. The reason it worked on my other computer was all the sounds were already cached on that machine then I installed the vpn afterwards. Thanks for your patience.

FYI the test sound is missing the .mp3 file extension still

![image](https://github.com/Hanabi-Live/hanabi-live/assets/73505906/44620e07-ed67-4a06-84a7-f65aa4f7080f)
